ee/tabby-ui/components/chat-list.tsx

```diff
@@ -27,6 +32,12 @@ export function ChatList({ messages, handleMessageAction }: ChatList) {
           )}
         </div>
       ))}
+      {isStreamResponsePending && (
+        <>
+          <Separator className="my-4 md:my-8" />
+          <MessagePendingIndicator />
```

Comment:
should this use a skeleton placeholder for consistency?

Reply:
<img width="513" alt="image" src="https://github.com/TabbyML/tabby/assets/17516233/499e6b75-eded-4222-8fa6-3ed9d503439c">